1.8/usage/tutorials/index.md

```diff
@@ -2,4 +2,39 @@
 post_title: Tutorials
 menu_order: 90 
 ---
-This is a collection of tutorials about using DC/OS. Learn how to run services and operate services in production.
+
+This is the DC/OS tutorials section, covering how to use DC/OS components as well as stateless and stateful DC/OS services. Typically, the services discussed here are available and installable from the [Universe](https://github.com/mesosphere/universe).
+
+The left-hand side menu lists the available basic tutorials, such as how to create and run a service. Further, there are a number of examples of how to use DC/OS services such as Spark or WordPress, which are available via the [dcos/examples](https://github.com/dcos/examples/) GitHub repo.
```

Comment:
`The tutorials that are located in [dcos/examples](https://github.com/dcos/examples/) GitHub repo are the most up-to-date. These tutorials are guaranteed to work the latest [versions](https://dcos.io/releases/) of DC/OS.`